tests/routers/test_encryption.py

```
@@ -51,3 +62,129 @@ def test_delete_key_not_found(self, authenticated_client):
+        initial_encrypted = notifier.smtp_password_encrypted
+        assert initial_encrypted is not None
+
+        # Rotate server key
+        response = authenticated_client.post("/api/v1/encryption/keys")
+        assert response.status_code == 200
+
+        # Query notifier again using the stored ID to avoid DetachedInstanceError
+        notifier = db_session.query(Notifier).filter_by(id=notifier_id).first()
+
+        # Check decryption works with the new key automatically
+        assert notifier.smtp_password == "my_secure_password"
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "smtp_password"

(S105)